### ISSUE TITLE
Notification generation for document expiry

### ISSUE THREAD

**Issue by pinarpink** [elixir-luxembourg/daisy#69]
Document entity should be extended with an expiry date field. And notifications should be generated on nearing expiry dates.

**Comment by vildead** [comment]
Dear @pinarpink @neoflex , 
document expiry is present, but when I try to store the document (Submit), I am getting nginx/1.12.2 error: 500 Internal Server Error

Daisy Log:
```
DEBUG 2019-08-06 13:52:49,476 utils 26040 140223987201856 uploading document post=<QueryDict: {}> files=<MultiValueDict: {}>
DEBUG 2019-08-06 13:52:49,478 utils 26040 140223987201856  submit_url=/documents/1/28/add
```
It seems that the problem rises on nginx level.

**Comment by pinarpink** [comment]
@vildead @neoflex I shall have a look locally. I had committed a fix with the notification branch.

**Comment by pinarpink** [comment]
@vildead @neoflex I cannot reproduce this locally. The query dicts should not be empty. Maybe a javascript issue, did you do "npm ci" for the update

**Comment by vildead** [comment]
closing the issue. Previous error solved in commit 1c02189752dc8519d0409685a0b2cdd319c95c96.
Notification functionality verified on: 12ccf4f02d13a91737e01dab3d1ff443f1bdbc8e
